ddl/db_test.go

```diff
@@ -3990,7 +4044,7 @@ func (s *testDBSuite4) TestComment(c *C) {
 	c.Assert(tk.Se.GetSessionVars().StmtCtx.WarningCount(), Equals, uint16(1))
 	tk.MustQuery("show warnings").Check(testutil.RowsWithSep("|", "Warning|1688|Comment for index 'e' is too long (max = 1024)"))
```

Comment:
Please add a test case for table partitions as well for completeness, something like:
```
"CREATE TABLE t (a int) PARTITION BY RANGE (a) (PARTITION p0 VALUES LESS THAN (0) COMMENT '" + valid/invalidComment + "')"
```
So valid + invalid for default sql_mode (start) and valid + invalid for sql_mode = ''.

Possibly also an ALTER, like:
```
ALTER TABLE t ADD PARTITION (PARTITION p1 VALUES LESS THAN (1000000) COMMENT '" + valid/invalidComment + "')"
```

Reply:
Thanks! 
Please merge this PR with [#1464](https://github.com/pingcap/tidb-test/pull/1464) to pass the mysqltest